lib/kb_Metrics/metricsDBs.py

```diff
@@ -87,133 +84,124 @@ def update_activity_records(self, upd_filter, upd_data):
             # return an instance of UpdateResult(raw_result, acknowledged)
             update_ret = self.mt_coll.update_one(upd_filter,
                                                  upd_op, upsert=True)
-        except BulkWriteError as bwe:
-            # pprint(bwe.details['writeErrors'])
+        except WriteError as we:
```

Comment:
How come your test passes while WriteError is undefined?

Reply:
due to an overriding mistake in the test file, fixed now.